src/FsAutoComplete.Core/CompilerServiceInterface.fs

```diff
@@ -25,7 +27,7 @@ type FSharpCompilerServiceChecker(backgroundServiceEnabled, hasAnalyzers) =
 
   // we only want to let people hook onto the underlying checker event if there's not a background service actually compiling things for us
   let safeFileCheckedEvent =
-    if not backgroundServiceEnabled then checker.FileChecked else (new Event<string * obj option>()).Publish
+    if not backgroundServiceEnabled then checker.FileChecked else (new Event<string * FSharpProjectOptions>()).Publish
```

Comment:
the change to the typed project options here comes because proj-info doesn't have the `obj` payload for extra data anymore, for better or worse